api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -256,35 +256,37 @@ def dispense(
         push_out: Optional[float] = None,
     ) -> InstrumentContext:
         """
-        Dispense a volume of liquid (in microliters/uL) using this pipette
-        into the specified location.
-
-        If only a volume is passed, the pipette will dispense from its current
-        position. If only a location is passed (as in
-        ``instr.dispense(location=wellplate['A1'])``), all of the liquid
-        aspirated into the pipette will be dispensed (this volume is accessible
-        through :py:attr:`current_volume`).
-
-        :param volume: The volume of liquid to dispense, in microliters. If 0
-                       or unspecified, defaults to :py:attr:`current_volume`.
+        Dispense liquid from a pipette tip. Includes parameters that control
+        dispense volume, well location, tip position in a well, and pipette
+        flow rate. See also, :ref:`new-dispense`.
+
+        :param volume: The volume to dispense, measured in µL.
+                        If 0 or unspecified, defaults to
+                        :py:attr:`current_volume`.
+                        If only a volume is passed, the pipette
+                        will dispense from its current position.
         :type volume: int or float
 
-        :param location: Where to dispense into. If `location` is a
-                         :py:class:`.Well`, the robot will dispense into
-                         :py:obj:`well_bottom_clearance.dispense <well_bottom_clearance>`
-                         mm above the bottom of the well. If `location` is a
-                         :py:class:`.Location` (i.e. the result of
-                         :py:meth:`.Well.top` or :py:meth:`.Well.bottom`), the
-                         robot will dispense into the exact specified location.
-                         If unspecified, the robot will dispense into the
-                         current position.
-        :param rate: A relative modifier for how quickly to dispense liquid.
-                     The flow rate for this dispense will be
-                     `rate` * :py:attr:`flow_rate.dispense <flow_rate>`.
-                     If not specified, defaults to 1.0.
+        :param location: Where to dispense liquid held in the pipette.
+                        If ``location`` is a :py:class:`.Well`,
+                        the robot will dispense into
+                        :py:obj:`well_bottom_clearance.dispense <well_bottom_clearance>`
+                        mm above the bottom of the well.
+                        If ``location`` is  :py:class:`.Location`
+                        (i.e. the result of :py:meth:`.Well.top` or
```

Comment:
```suggestion
                        (e.g., the result of :py:meth:`.Well.top` or
```
this list is non-exhaustive. and a comma. 🤓